tests/gaudi/l2/README.md

```diff
@@ -75,3 +75,104 @@ Welcome to HCCL demo
 [BENCHMARK]     Algo Bandwidth : 147.548069 GB/s
 ####################################################################################################
 ```
+<<<<<<< HEAD
```

Comment:
PR git comments: the buildconfig base on https://github.com/HabanaAI/vllm-fork/pull/602

Reply:
will add these